root.tf

```diff
@@ -206,3 +213,18 @@ module "waf" {
   geo_match         = split(",", var.geo_match)
   restricted_uri    = "auth/admin"
 }
+
+module "backend_lambda_function_bucket" {
+  source = "./tdr-terraform-modules/s3"
+  common_tags = local.common_tags
+  function = "backend-checks"
+  project = var.project
+}
+
+module "av_lambda" {
```

Comment:
Would you mind renaming this from "av" to "antivirus" so it's clear to someone who's not familiar with what backend checks TDR does? I always think "audiovisual" before I remember it stands for "antivirus".

Reply:
Yeah sure, I've renamed it.